synapse/tests/test_cmds_cortex.py

```diff
@@ -55,90 +53,89 @@ def test_storm(self):
             outp.expect("took")
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm teststr=b')
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm teststr=b')
             outp.expect('complete. 0 nodes')
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm teststr=abcd')
-            outp.expect(':tick = 1970/01/01 00:00:00.123')
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm teststr=abcd')
+            outp.expect(':tick = 2015/01/01 00:00:00.000')
             outp.expect('#cool')
             outp.expect('complete. 1 nodes')
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm --hide-tags teststr=abcd')
-            outp.expect(':tick = 1970/01/01 00:00:00.123')
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm --hide-tags teststr=abcd')
+            outp.expect(':tick = 2015/01/01 00:00:00.000')
             self.false(outp.expect('#cool', throw=False))
             outp.expect('complete. 1 nodes')
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm --hide-props teststr=abcd')
-            self.false(outp.expect(':tick = 1970/01/01 00:00:00.123', throw=False))
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm --hide-props teststr=abcd')
+            self.false(outp.expect(':tick = 2015/01/01 00:00:00.000', throw=False))
             outp.expect('#cool')
             outp.expect('complete. 1 nodes')
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm --hide-tags --hide-props teststr=abcd')
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm --hide-tags --hide-props teststr=abcd')
             self.false(outp.expect(':tick = 1970/01/01 00:00:00.123', throw=False))
             self.false(outp.expect('#cool', throw=False))
             outp.expect('complete. 1 nodes')
 
             outp = self.getTestOutp()
-            cmdr = s_cmdr.getItemCmdr(core, outp=outp)
-            cmdr.runCmdLine('storm --raw teststr=abcd')
-            outp.expect("'tick': 123")
+            cmdr = await s_cmdr.getItemCmdr(core, outp=outp)
+            await cmdr.runCmdLine('storm --raw teststr=abcd')
```

Comment:
this does not validate teh raw data was printed; just teh repr form

Reply:
This test should validate the raw timestamp, not the repr'd timestamp; since the `--raw` option is passed to storm

Reply:
got it